src/components/search-form/search-form.test.js

```diff
@@ -0,0 +1,79 @@
+import { MemoryRouter } from 'react-router-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { SearchForm } from './search-form';
+import { useGetParams } from '../../hooks/use-get-params/use-get-params';
+
+jest.mock('../../hooks/use-get-params/use-get-params', () => ({
+  useGetParams: jest.fn(() => [{}]),
+}));
+
+describe('Search input', () => {
+  it('Matches the snapshot', () => {
+    render(
+      <MemoryRouter>
+        <SearchForm />
+      </MemoryRouter>
+    );
+
+    const searchInputElement = screen.getByRole('searchbox');
+
+    expect(searchInputElement).toMatchSnapshot();
+  });
+
+  it('Can contain a value', async () => {
+    render(
+      <MemoryRouter>
+        <SearchForm />
+      </MemoryRouter>
+    );
+
+    const inputValue = 'the';
+    const searchInputElement = screen.getByRole('searchbox');
+
+    await userEvent.type(searchInputElement, inputValue);
+
+    waitFor(() => expect(searchInputElement).toHaveValue(inputValue));
+  });
+})
+
+describe('Search button', () => {
+  it('Matches the snapshot', () => {
+    render(
+      <MemoryRouter>
+        <SearchForm />
+      </MemoryRouter>
+    );
+
+    const searchButtonElement = screen.getByRole('button');
+
+    expect(searchButtonElement).toMatchSnapshot();
+  });
+})
+
+describe('Search Form', () => {
+  it('Handles input value on button click', async () => {
```

Comment:
On what button click? Please be more specific. I think it should be on the search button click.

Reply:
Changed it to `'Handles input value on search button click'`.